.vib/clickhouse/goss/clickhouse.yaml

```diff
@@ -0,0 +1,29 @@
+command:
+  check-clickhouse-server:
+    exec: timeout --preserve-status 5 clickhouse-server 2>&1 || true
+    exit-status: 0
+    stdout:
+      - Ready for connections
+file:
+  /var/lib/clickhouse/tmp:
+    exists: true
+    filetype: symlink
+    linked-to: /opt/bitnami/clickhouse/tmp
+  /opt/bitnami/clickhouse/logs/clickhouse.log:
+    exists: true
+    filetype: symlink
+    linked-to: /dev/stdout
+  /opt/bitnami/clickhouse/etc/config.xml:
```

Comment:
I would also check that `users.xml` is updated properly to pick up the password from the env var:

https://github.com/bitnami/containers/blob/435b7670131950bc31c7b373b796efe471434146/bitnami/clickhouse/22.8/debian-11/rootfs/opt/bitnami/scripts/clickhouse/postunpack.sh#LL60C144-L60C153